staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -715,6 +717,7 @@ type UpdateOptions struct {
 // Preconditions must be fulfilled before an operation (update, delete, etc.) is carried out.
 type Preconditions struct {
 	// Specifies the target UID.
+	// More info: https://kubernetes.io/docs/user-guide/identifiers/#uids
```

Comment:
Why are we adding new entries to the `/user-guide/` path which has been removed for 5 years.

Reply:
I agree: best not to.

Reply:
Try this: https://k8s.io/docs/concepts/overview/working-with-objects/names/#uids

Reply:
Sry, I guess I misunderstood the meaning of liggitt, I should use the latest URL here.